docs/UserGuide.md

```diff
@@ -191,6 +191,10 @@ This section details the format of the commands available in TutorBuddy. We will
 
 * Date must be input in the format, `yyyy-mm-dd`.
   e.g. `2021-01-01` is allowed, but not `01-01-2021`.
+  
+* Year must be between 1970 and 2037, both inclusive.
```

Comment:
Maybe we want to include the rationale of why we do it this way, (To avoid the 2038 problem which can found [here](https://en.wikipedia.org/wiki/Year_2038_problem))